Graphs/test/KahnsAlgorithm.test.js

```diff
@@ -0,0 +1,13 @@
+import { KahnsAlgorithm } from '../KahnsAlgorithm'
+
+test('Graph without cycle', () => {
```

Comment:
As said, these two `test` blocks should be in a `describe("Kahn's algorithm", () => {...})` block.